akka-stream-tests/src/test/scala/akka/stream/impl/fusing/InterpreterSpec.scala

```diff
@@ -564,4 +567,51 @@ class InterpreterSpec extends InterpreterSpecKit {
 
   }
 
+  private[akka] case class Doubler[T]() extends PushPullStage[T, T] {
```

Comment:
This is just a move from InterpreterSpecKit, but I have not yet removed it from there.